src/Integration.Vsix/SonarLintTagger/IScheduler.cs

```diff
@@ -31,6 +31,6 @@ public interface IScheduler
         /// <param name="jobId">Unique key based on which previous job is cancelled</param>
         /// <param name="action">Job action</param>
         /// <param name="timeoutInMilliseconds">number of milliseconds after which the job should be cancelled</param>
-        void Schedule(string jobId, Action<CancellationToken> action, int timeoutInMilliseconds);
+        void Schedule(string jobId, Action<CancellationToken> action, int? timeoutInMilliseconds);
```

Comment:
@duncanp-sonar I changed the parameter to nullable, since the tests started to flake. I suspect that this is because CancelAfter triggers the timer anyway, even if you pass Timeout.Inifinite. Since that's what we pass for JS, I decided to change that to null, so it will be explicit that a timer is triggered only when a value is deliberately passed.